src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -1347,6 +1350,16 @@ void BlockchainLMDB::open(const std::string& filename, const int db_flags)
     throw DB_ERROR("Database could not be opened");
   }
 
+#ifdef WIN32
+  // ensure NTFS compression is disabled on the directory and database file to avoid corruption of the blockchain 
+  if (boost::filesystem::exists(filename) && !disable_ntfs_compression(filename)) 
+    LOG_PRINT_L0("Failed to disable NTFS compression on folder: " << filename << ". Error: " << ::GetLastError());
+  boost::filesystem::path datafile(filename);
+  datafile /= CRYPTONOTE_BLOCKCHAINDATA_FILENAME;
+  if (boost::filesystem::exists(datafile) && !disable_ntfs_compression(datafile)) 
```

Comment:
I would touch the file so that it always exists, otherwise the daemon might have a compressed db on first launch. Also, if disabling compression fails, I'd throw a DB_ERROR.